lms/templates/problem.html

```diff
@@ -47,8 +47,9 @@ <h3 class="hd hd-2 problem-header">
     % endif
     % if attempts_allowed :
     <div class="submission_feedback" aria-live="polite">
-      ${_("You have used {num_used} of {num_total} submissions").format(num_used=attempts_used, num_total=attempts_allowed)}
+      ${_("You have used {num_used} of {num_total} attempts").format(num_used=attempts_used, num_total=attempts_allowed)}
     </div>
     % endif
+    <%include file="problem_notifications.html" args="notification_type=notification_type,notification_icon=notification_icon,notification_message=notification_message,hint_notification=hint_notification"/>
```

Comment:
For the purposes of this story, comment out this line so that the dummy "hint" notification is not displayed.